nucliadb/nucliadb/ingest/service/writer.py

```diff
@@ -777,9 +777,12 @@ async def ReIndex(self, request: IndexResource, context=None) -> IndexStatus:  #
                 await kbobj.set_resource_shard_id(request.rid, shard.sharduuid)
 
             if shard is not None:
-                logger.info("Calling shard.add_resource")
-                counter = await shard.add_resource(brain.brain, 0, uuid.uuid4().hex)
-                logger.info("Finished shard.add_resource")
+                counter = await shard.add_resource(
+                    brain.brain,
+                    0,
+                    partition="reindex",
```

Comment:
@bloodbare any objections with this? So if we ever do a reindex, we could subscribe to this partition to check when it has finished.

Reply:
Feel the partition Luke